package.json

```diff
@@ -34,21 +34,14 @@
     "disableAutoBuild": 1
   },
   "activationEvents": [
-    "onCommand:quarkusTools.createProject",
-    "onCommand:quarkusTools.welcome",
     "workspaceContains:**/src/main/resources/application.properties",
     "workspaceContains:**/src/main/resources/application-?*.properties",
     "workspaceContains:**src/main/resources/META-INF/microprofile-config.properties",
     "workspaceContains:**/src/main/resources/application.yaml",
     "workspaceContains:**/src/main/resources/application.yml",
     "workspaceContains:**/src/main/resources/templates",
     "onLanguage:microprofile-properties",
-    "onLanguage:quarkus-properties",
-    "onLanguage:java",
-    "onLanguage:qute-html",
-    "onLanguage:qute-json",
-    "onLanguage:qute-yaml",
-    "onLanguage:qute-txt"
```

Comment:
onCommand and onLanguage events are automatically registered, no need to declare them explicitly here